### PR TITLE
Fix live code example width

### PR DESCRIPTION
**Pull request by colebemis**
| Before | After |
| --- | --- |
| image image |  image image |

### CODE REVIEW

File: docs/src/@primer/gatsby-theme-doctocat/components/live-preview-wrapper.js [hunk 1]
```diff
@@ -5,9 +5,11 @@ import React from 'react'
...
 function LivePreviewWrapper({children}) {
   return (
-    <BaseStyles>
-      <Box p={3}>{children}</Box>
-    </BaseStyles>
+    <Box width="100%">
+      <BaseStyles>
+        <Box p={3}>{children}</Box>
+      </BaseStyles>
+    </Box>
```
Comment (colebemis) [inline docs/src/@primer/gatsby-theme-doctocat/components/live-preview-wrapper.js:12]:
```suggestion
    <Box width="100%" p={3}>
      <BaseStyles>{children}</BaseStyles>
    </Box>
```

**Comment by T-Hugs** [comment]
This kind of seems like a band-aid to fix the [real issue](https://github.com/primer/doctocat/blob/master/theme/src/components/live-code.js#L50), which is the parent Flex container using `row` as `flex-direction`. In my opinion, the parent shouldn't be a Flexbox at all - just a regular block element.

**Review by bscofield — APPROVED** [review]
👍

**Comment by bscofield** [comment]
(ah sorry, saw @T-Hugs comment after approving the band-aid -- I don't have a strong preference for his suggested fix vs. this one)

**Review by emplums — APPROVED** [review]

**Comment by emplums** [comment]
🤦‍♀️ I too missed Trevor's comment before approving and merging (was quickly trying to clean up our PRs ahead of today's release). Sorry about that! @colebemis if you would like to make the flex box change feel free and I'll review that separately

**Pull request merged**